python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -471,10 +494,11 @@ def __set__(self, obj, val):
         new_value = Descriptor.cleanValue(obj, val, _set_name)
 
         obj.setSchemaAttribute(_set_name, new_value)
+
         obj._setDirty()
 
-        if isinstance(val, Node):
-            val._setDirty()
+        # This is currently disabled due to bad interaction with the GangaDirac tests
+        #Descriptor.check_inheritance(obj, _set_name)
```

Comment:
I was getting errors due to non parented objects shown in this PR which exposed a horrible screwup elsewhere in the codebase. I can revert this but it's a nice realtime test to enable at some point so I'll drop it to another PR if it's felt it's not needed.